api/base.proto

```diff
@@ -149,3 +150,16 @@ message ConfigSource {
     AggregatedConfigSource ads = 3;
   }
 }
+
+// Configuration for low level socket in listeners and clusters. If the configuration is empty,
+// the default low level socket configuration will be choosed based on the platform and
```

Comment:
s/choosed/chosen/
s/the default/a default/
s/low level/low-level/

Reply:
thanks :)